linux/jdk/debian/src/main/packaging/temurin/17/debian/changelog

```diff
@@ -1,3 +1,9 @@
+temurin-17-jdk (17.0.4.0.0+8) UNRELEASED; urgency=medium
```

Comment:
```suggestion
temurin-17-jdk (17.0.4.0.0+8) stable; urgency=medium
```

Looks like we should have switched this to stable some time ago. Let's do it from this release onwards.